src/components/KBreadcrumbs/KBreadcrumbs.vue

```diff
@@ -6,44 +6,36 @@
     <li
       v-for="(item, idx) in items"
       :key="getBreadcrumbKey(item, idx)"
-      class="k-breadcrumbs-item"
+      class="breadcrumbs-item-container"
     >
       <component
-        :is="getComponentAttrs(item).type"
-        v-bind="getComponentAttrs(item).attrs"
-        class="no-underline"
+        :is="getComponentAttrs(item, idx === items.length - 1).type"
```

Comment:
**Issue**: I just noticed that the behavior of KBreadcrumbs was changed to only make non-last items links. For how we use breadcrumbs, that’s a regression because the last breadcrumb is something you want to be able to navigate to (arguably the most important breadcrumb). I noticed it because we have tests for this sort of navigation that break in my adoption PR (https://github.com/kumahq/kuma-gui/pull/2344).

For example, in gateway manager, when I’m in the Data Plane Nodes view, I want to be able to click the name of the runtime group (e.g. default) to go back to its overview.

Reply:
FIxing PR: https://github.com/Kong/kongponents/pull/2098